packages/atomic/cypress/integration/common-assertions.ts

```diff
@@ -3,7 +3,7 @@ import {AriaLiveSelectors} from './aria-live-selectors';
 import {ComponentErrorSelectors} from './component-error-selectors';
 
 export interface ComponentSelector {
-  shadow: () => Cypress.Chainable<JQuery<HTMLElement>>;
+  shadow: () => Cypress.Chainable<JQuery<any>>;
```

Comment:
Why the change to any ?

Reply:
which is weird since HTMLAtomicCategoryFacetElement extends HTMLStencilElement which in turn extends HTMLElement